src/core/immersed_boundary/ImmersedBoundaries.cpp

```diff
@@ -312,8 +313,8 @@ void ImmersedBoundaries::calc_volume_force() {
              vector_product(x2, x1, n);
              for (int k=0; k < 3; k++) p3.f.f[k] += fact*n[k];*/
 
-            // This is Dupin 2008. I guess the result will be very similar as
-            // the code above
+            // This is eq. (9) in @cite dupin08a. I guess the result will be
```

Comment:
Is this seriously a in-code documentation comment: "I guess the result will be very similar as the code above" ??? + missing punctuation.

Reply:
This file has 20% coverage. IBM is essentially unmaintained. It has been a candidate for downgrade or removal for more than a year (#2071, #2305).